src/main/java/ama/post/PostController.java

```diff
@@ -100,5 +100,22 @@ public String downvoteComment(HttpServletRequest request, @RequestParam("id") St
         }
         return "pageNotFound";
     }
+
+    @PostMapping("/comments/{id}/reply")
+    public String replyToComment(HttpServletRequest request, @PathVariable(value="id") String id, String reply) {
+        CommentPost comment = commentPostRepository.findById(Long.parseLong(id)).get(0);
```

Comment:
'findById()' by design returns a list, but since id's are unique, we can assume there is only one comment in that list, which is at index 0.